submissions/Jigsaff/dom-api-task/app/app.js

```diff
@@ -0,0 +1,147 @@
+        name: "TREKKING",
+        img: "./img/trekking.webp",
+        description: [
+            "Our Aircontact trekking backpack has won the title of “indestructible”. Therefore, we have opted for gentle changes: It is extremely robust, but now it is even neater, more modern and streamlined when trekking. And even more comfortable: Thanks to the new, flexible, suspended Active Fit shoulder straps and the re-designed hip fins, the backpack moulds itself comfortably around the wearer – for noticeably more comfort. For women, the new system is equipped with the tried and tested features of the SL fit."
+        ],
+    },
+    {
+        name: "HIKING",
+        img: "./img/hiking.webp",
+        description: [
+            "The Fox, our trekking backpack for kids and teenagers, was revised to provide the technical design and great functionality of our “big” backpacks fully adapted to the needs of kids. Thanks to the VariQuick back length adjustment system, the Fox also grows with the child so they can enjoy with their adventure backpack for a longer time."
+        ],
+    },
+    {
+        name: "CHILD CARRIER",
+        img: "./img/child-carrier.webp",
+        description: [
+            "The new, compact Lite Air contact back with ventilation makes this child carrier absolutely lightweight. The sporty frame offers the height of comfort and naturally bears the TÜV GS safety seal of approval. The seating area for the child is a comfort zone with the simple-to-use and easy-to-access safety harness, the side access and the very soft cover material. The unisex fit with specially adapted shoulder straps and hip fins lets mum and dad take turns carrying the child."
+        ],
+    },
+];
+
+const headerUl = document.querySelector(".header__ul");
+const mainHeader = document.querySelector(".main__header");
+const mainImage = document.querySelector(".main__img");
+const mainDescription = document.querySelector(".main__description");
+
+const generateDescription = (section) => {
+    const fragmentDescription = document.createDocumentFragment();
+
+    section.description.forEach((description) => {
+        const desc = document.createElement("p");
+        desc.textContent = description;
+        desc.classList.add("main__description-p");
+        fragmentDescription.appendChild(desc);
+    });
+    mainDescription.innerHTML = "";
+    mainDescription.appendChild(fragmentDescription);
+};
+
+const handleHeaderLinkClick = (target) => {
+    const activeElement = document.querySelector(".header__link.header__link--decorated");
+
+    if (activeElement) {
+        activeElement.classList.remove("header__link--decorated");
+    }
+    target.classList.add("header__link--decorated");
+};
+
+const handleHeaderClick = ({target}) => {
+    if (target.classList.contains("header__link")) {
+        const sectionName = target.textContent;
+        const section = backpacks.find(
+            (item) => item.name.toLowerCase() === sectionName.toLowerCase()
```

Comment:
You can destructure your `item` object here. Also, I'd recommend using not so generic names for variables. Using the `item` variable name is not very informative